app/controllers/rules_of_use_controller.rb

```diff
@@ -0,0 +1,49 @@
+class RulesOfUseController < ApplicationController
```

Comment:
WDYT of namespaceing this under `users/` ? Since it's a sort-of logged in thing?